File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        slow=0
        count=1 #number of elements
        for fast in range(len(nums)):
            if nums[fast]!=nums[slow]:
                slow+=1
                count+=1
                nums[slow]=nums[fast]
        
        return count
                
                
            
           

File: 26-remove-duplicates-from-sorted-array/26-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int start = 0;
        for(int end = 1;end<nums.size();end++){
            if(nums[end] != nums[start]){
                swap(nums[end], nums[start+1]);
                start ++;
            }
        }
        return start+1;
    }
};